components/widgets/inputs/dropdown_input_hybrid.scss

```diff
@@ -36,12 +36,21 @@
     }
 
     .Input_fieldset {
+        position: relative;
         display: flex;
         padding: 0;
     }
 
     .Input_legend {
+        position: absolute;
+        z-index: 1;
```

Comment:
We could probably, at some point, reuse rules from the input, so rules like the one below do match, but it's no biggie for this case.

https://github.com/mattermost/mattermost-webapp/blob/aac1dc5fb5a46d72db6d0e83d8dc3df95d5e48f6/components/widgets/inputs/input/input.scss#L168